### PR TITLE
Phrase1 TeamCity

### PR DESCRIPTION
**Pull request by phanthaoIT**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (lilyhdo) [inline .gitignore:2]: remove this file

File: 1-trigger-TeamCity.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    >+ Copy "Homepage URL" and "authorization callback URL" in the "Add Connection" dialog. 
+
+![alt text](./assets/OAuth.png )
+
++ A page with "Client ID" and "Client Secret" will open. Copy and paste them to TeamCity and click `Save`. 
+
+![alt text](./assets/get_client.png )
+
+![alt text](./assets/save_connect.png )
+## 1.3 Creat a project
```
Comment (lilyhdo) [inline 1-trigger-TeamCity.md]: Creat -> Create

File: 1-trigger-TeamCity.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Triggering a Teamcity build from a push GitHub
+
+This guide will demonstrate how to get Teamcity to build your project every time your soure code is changed and pushed to GitHub.
```
Comment (lilyhdo) [inline 1-trigger-TeamCity.md]: soure -> source

File: 1-trigger-TeamCity.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
++ TeamCity account
++ GitHub repository
+## 1.1. Sign in to Teamcity 
++ You can download and install TeamCity from [here](https://confluence.jetbrains.com/display/TCD10/Installation+Quick+Start).  
+
++ Start TeamCity server, enter your TeamCity username and password, then click `Log in`.
+
+![alt text](./assets/login.png )
+## 1.2 Connect TeamCity with GitHub
++ Now you need to creat a project, to do this click on `Projects` and then `Creat Project`.
```
Comment (lilyhdo) [inline 1-trigger-TeamCity.md]: Creat -> Create

File: 2-run-automation-test.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+  udid:               {your_device_udid}
+}
+```
+
+**b. Configure in your TeamCity project**
++ In your project in TeamCity, on the left side on the page, click on 'Build Step' to set build steps.
++ On the right side of the page,  for the 'Runner type' category, choose 'Command Line '.
++ Add command lines to your script content. 
+
+Example:
```
Comment (thangvo38) [inline 2-run-automation-test.md]: `cd javascript` is unnecessary for this example and might confuse the readers

File: 2-run-automation-test.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+For more information, go to https://api.kobiton.com/docs/
+
+**Get session Commands**
+```
+  GET /session/{sessionId}/commands
+```
+To get a certain page of your commands, add page parameter in your query.
+
+For example: GET /session/{sessionId}/commands?page=2
```
Comment (thangvo38) [inline 2-run-automation-test.md]: It should be:

For example:
`GET /session/{sessionId}/commands?page=2`

File: README.md [hunk 7]
```diff
@@ -1,2 +1,15 @@
 # TeamCity-TestFlight-Appium-ReactNative
 Guidance on integrating Kobiton service into the mobile app build pipeline: TeamCity, TestFlight, Appium and ReactNative.
+
+If you are using:
++ TeamCity to build app
++ TestFlight for keeping the daily app build
++ Appium to write the automation test
++ Reactnative to write app
+
+The guidelines in this repo will show you how to trigger a TeamCity build from a push on GitHub, and how to run automation tests with Kobiton.
```
Comment (thangvo38) [inline README.md]: Please write something to let the readers know that Kobiton supports Appium. This doesn't highlight the importance of Kobiton in a pipeline.

File: 1-trigger-TeamCity.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
+# Triggering a Teamcity build from a push GitHub
```
Comment (thangvo38) [inline 1-trigger-TeamCity.md:1]: Is this document necessary? I think we already assume the readers know how to do this step.

**Review by thangvo38 — CHANGES_REQUESTED** [review]
Request changes

File: 1-trigger-TeamCity.md (hunk 8 above)
Comment (phanthaoIT) [reply 1-trigger-TeamCity.md:1]: i think it is no problem

**Pull request closed without merging**